plugins/inventory/iocage.py

```diff
@@ -201,11 +201,10 @@ def get_inventory(self, path):
             try:
                 t_stdout = to_text(stdout, errors='surrogate_or_strict')
             except UnicodeError as e:
-                raise AnsibleError('Invalid (non unicode) input returned: %s' % to_native(e)) from e
+                raise AnsibleError(f'Invalid (non unicode) input returned: {e}') from e
```

Comment:
Lines 198-199 above were clearly missed by the automated tool, they might be rewritten as an f-str and another `to_native()` would disappear.

Reply:
The inventory plugin iocage was only merged after your tool ran :)

Reply:
8353cb6967308f3d2d96c2b363dceae06d6384d4